src/components/savings/SavingsListRow.js

```diff
@@ -166,7 +166,7 @@ const SavingsListRow = ({
             as={android && ButtonPressAnimation}
             css={padding(9, 10, 10, 11)}
             justify="space-between"
-            onPress={NOOP}
+            onPress={android ? onButtonPress : NOOP}
```

Comment:
```suggestion
            onPress={onButtonPress}
```

I this it's sufficient, cause `Row` does not have onPress